templates/module.rs.template

```diff
@@ -36,12 +36,13 @@ impl #module_name {
 
 #[cfg(test)]
 mod tests {
-    use super::#module_nameDeployer;
+    use super::*;
+    use odra::host::{Deployer, NoInit};
 
     #[test]
     fn it_works() {
         let env = odra_test::env();
-        let mut contract = #module_nameDeployer::init(&env);
+        let mut contract = #module_nameHostRef::deploy(&env, NoInit);
```

Comment:
```suggestion
        let mut contract = #module_nameHostRef::deploy(&env, NoArgs);
```

Reply:
It's recommended to read the documentation first. :) 
My idea was `EmptyArgs` or `DefaultArgs`. Now both start with `No` that might be a source of confusion.

Reply:
Fixed